packages/frontend/src/components/common/balance/helpers.js

```diff
@@ -38,6 +39,16 @@ export const getRoundedBalanceInFiat = (rawNearAmount, tokenFiatValue) => {
     return roundedBalanceInFiat;
 };
 
+export const getTotalBalanceInFiat = (mainTokens) => {
+    const totalAmount = mainTokens.map((el) => {
+    const USD = el.coingeckoMetadata.usd
+    const balance = el.balance
+      return el.contractName ? getRoundedBalanceInFiat(balance,USD,true,el.onChainFTMetadata.decimals) : getRoundedBalanceInFiat(balance,USD)
+    }).reduce((a,b) =>`${+a + +b}`)
+    
+    return totalAmount !== NaN ? new Intl.NumberFormat("en-EN",{maximumFractionDigits:2,minimumFractionDigits:2}).format(totalAmount) :'0'
```

Comment:
similar comment re connecting to the codes we have in `state.localize.languages` based on the currently active language